theano/sandbox/cuda/opt.py

```diff
@@ -1904,8 +1904,7 @@ def local_gpu_downsample_factor_max_grad(node):
             return
         x, z, gz = node.inputs
         if (x.owner and isinstance(x.owner.op, HostFromGpu)):
-            gpu_ds_grad = GpuDownsampleFactorMaxGrad(node.op.ds,
-                                                     node.op.ignore_border)
+            gpu_ds_grad = GpuDownsampleFactorMaxGrad(**node.op._props_dict())
```

Comment:
idem. The node.op here have more props then the GPU ops. So this will fail. Revert this. The assert at line 1898 will safeguard us.